projects/driver_controls/test/test_mech_brake_calibration.c

```diff
@@ -0,0 +1,69 @@
+#include "ads1015.h"
```

Comment:
Please add a comment explaining that this isn't actually a test and how it's intended to be used.